src/mesh/README.rst

```diff
@@ -9,7 +9,7 @@ Commands to manage Azure Service Fabric Mesh resources
         az mesh: Manage Azure Service Fabric Mesh resources.
 
```

Comment:
https://docs.microsoft.com/en-us/azure/service-fabric-mesh/ Set this link to our official doc